canonicalwebteam/blog/wordpress.py

```diff
@@ -111,9 +111,13 @@ def get_article(self, slug, tags=None, tags_exclude=None):
         Get an article from Wordpress api
         :param slug: Article slug to fetch
         """
-        return self.get_first_item(
-            "posts", {"slug": slug, "tags": tags, "tags_exclude": tags_exclude}
-        )
+        try:
```

Comment:
Can we move the `try` block into the views and `flask.abort` with the appropriate codes?

Reply:
Actually nevermind.